sos/report/plugins/selinux.py

```diff
@@ -51,7 +51,8 @@ def setup(self):
                 'login',
                 'node',
                 'interface',
-                'module'
+                'module',
+                'boolean'
```

Comment:
minor nit, but not a blocker, can you add a comma at the end

Reply:
After 'boolean'? I guess that's last member or list. No need of comma. Correct me if I am wrong

Reply:
There's no need, but when adding new entries it just one line change, rather than 2, hence it's nice to have that.

I don't mind if you don't add it, it's just a preference.